nanopub/nanopub.py

```diff
@@ -200,114 +207,111 @@ class NanopubClient:
     """
     def __init__(self):
         self.java_wrapper = JavaWrapper()
+        self.grlc_urls = NANOPUB_GRLC_URLS
 
-    def search_text(self, searchtext, max_num_results=1000,
-                    apiurl='http://grlc.nanopubs.lod.labs.vu.nl//api/local/local/find_nanopubs_with_text'):
+    def find_nanopubs_with_text(self, text, max_num_results=1000):
```

Comment:
I think this renaming has been a good idea. It makes everything consistent with the grlc API.

Reply:
Great!